E3SM-1-0-LE/historical_r1i2p2f1.json

```diff
@@ -0,0 +1,103 @@
+{
+  "#note_source_type": "explanation of what source_type is goes here",
+
+  "source_type": "AOGCM AER",
+
+  "#note_experiment_id": "CMIP6 valid experiment_ids are found in CMIP6_CV.json",
+
+  "experiment_id": "historical",
+
+  "activity_id": "CMIP",
+
+  "sub_experiment_id": "none",
+
+  "realization_index": "1",
+
+  "initialization_index": "2",
+
+  "physics_index": "2",
```

Comment:
Advance to 2 for physics_index to differentiate from v1 original 5 historical runs